tests/DocBlock/TypeExpressionTest.php

```diff
@@ -372,6 +374,24 @@ public static function provideParseInvalidExceptionCases(): iterable
 
         yield ['((unclosed_parenthesis)'];
 
+        yield ['|vertical_bar_start'];
+
+        yield ['&ampersand_start'];
+
+        yield ['~tilde_start'];
```

Comment:
```suggestion
```

Reply:
what is the change here?

Reply:
it is completely fine to test this invalid case

Reply:
The suggestion just removes 2 lines 🙂. Support for subtraction types was reverted, so these cases are superfluous. `~` at this point does not have any special meaning, so there's no point in including them here. I know these are proper "invalid types" that's why tests are green, but then why we don't have cases like `!int` or `int!` 😉? 

It's not crucial, I just thought it's a leftover from cleaning-up the code.

Reply:
I prefer to keep these tests as they are valid now and will be later too. If someone else will implement `~` later, he will very likely not add these, so we add them now and do not have to worry later...

Reply:
Let's have it there, we can remove it later if needed. Currently I consider it a bad type/phpdoc type.